bugbear.py

```diff
@@ -229,3 +254,13 @@ def compose_call_path(self, node):
             "to the exception.",
     type=BugBearChecker,
 )
+
+B901 = partial(
+    error,
+    message=("B901: Using ``yield`` together with ``return x``. Use ``raise "
+             "StopIteration(x)`` instead of ``return x`` if this was "
+             "intentional."),
```

Comment:
Again, because of PEP 479, I'd :

"B901: Using ``yield`` together with ``return x`` is confusing to the reader. Use native ``async def`` coroutines or put a ``# noqa`` comment on this line if this was intentional."